recipe/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "seal" %}
+{% set version = "4.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/microsoft/SEAL/archive/refs/tags/v4.1.1.tar.gz
+  sha256: af9bf0f0daccda2a8b7f344f13a5692e0ee6a45fea88478b2b90c35648bf2672
+
+build:
+  number: 0
+  ignore_run_exports:
+    - gtest
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - zlib {{ zlib }}
+    - zstd {{ zstd }}
+    - ms-gsl 4.0.0
+    - gtest
+  run:
+    - ms-gsl 4.0.0
+    - zlib  # pin through run_exports
+    - zstd  # pin through run_exports
+
+test:
+  commands:
```

Comment:
`-DSEAL_BUILD_TESTS=ON` but you aren't running the tests?

Reply:
I do run them, inside the build itself (see https://github.com/AnacondaRecipes/seal-feedstock/pull/1/files#diff-d7075654874cb08007a21aaab3ecd4b3453a9087e7505d034d548b8938b599bcR9). I tried to run them here (like I did in https://github.com/AnacondaRecipes/xtl-feedstock/pull/1) but it was causing way too many problems.

Reply:
ok, thanks